teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1155 @@
+{
```

Comment:
<p>Add info to authors, see e.g. <a href="https://github.com/volkamerlab/teachopencadd/blob/t011-base/teachopencadd/talktorials/T012_query_klifs/talktorial.ipynb" target="_blank">T012</a> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>